app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -75,6 +75,15 @@ public void onCreate(@Nullable final Bundle savedInstanceState) {
                 .getPreferredContentCountry(requireContext());
         initialLanguage = PreferenceManager
                 .getDefaultSharedPreferences(requireContext()).getString("app_language_key", "en");
+
+        findPreference(getString(R.string.clear_cookie_key))
+                .setOnPreferenceClickListener(preference -> {
+                    defaultPreferences.edit()
+                            .putString(getString(R.string.recaptcha_cookies_key), "").apply();
+                    Toast.makeText(getActivity(), R.string.recaptcha_cookies_cleared,
+                            Toast.LENGTH_SHORT).show();
+                    return true;
```

Comment:
Here you should also clear the downloader's cookies:
`DownloaderImpl.getInstance().setCookie(ReCaptchaActivity.RECAPTCHA_COOKIES_KEY, "");`

Reply:
Oh, I forgot we kept it in memory there.